src/ts/view/index.ts

```diff
@@ -0,0 +1,34 @@
+import { EQUAL } from "../constants/index.js";
+import {
+  checkFirstTotalValue,
+  checkPreventClickValue,
+  isNumber,
+} from "../controller/validator.js";
+import { calculator, makeFormula } from "../model/calculator.js";
+
+export const clearTotalValue = (): void => {
+  document.getElementById("total")!.innerText = "0";
```

Comment:
```suggestion
  $("total")!.innerText = "0";
```
다른 코드 보니까 이렇게 간단하게 표현한 코드가 있어서 공유합니댜.

`const $ = (id) => document.getElementById(id)` 로 정의하면 위와같이 표현할 수 있습니다.

Reply:
넘 좋네요. 담 프로젝트에 한번 사용해보겠습니다.